### PR TITLE
Abstract the location of the kythe directory

### PR DESCRIPTION
**Pull request by mpickering**
This introduces a new variable KYTHE_DIR which specifies where
Kythe is installed.

### CODE REVIEW

File: serve.sh [hunk 1]
```diff
@@ -5,27 +5,29 @@ if (($# != 2)); then
 fi
 
 INDEXER_OUTPUT_DIR=$1
+KYTHE_DIR=/opt/kythe
```
Comment (robinp) [inline serve.sh]: Maybe only do this if it's not already set?

Comment (robinp) [inline serve.sh]: Could you also update the README.md with instructions to set it?

**Comment by mpickering** [comment]
How does that look?

**Comment by robinp** [comment]
Thank you! Merging.

**Pull request merged**